### PR TITLE
ko: update cheatsheet in dev-1.14-ko.4 

### PR DESCRIPTION
**Pull request by jmyung**
Fixed partially #14207 

- docs/reference/kubectl/cheatsheet.md

/language ko

### CODE REVIEW

File: content/ko/docs/reference/kubectl/cheatsheet.md [hunk 1]
```diff
@@ -265,6 +267,8 @@ kubectl delete pod,service baz foo                                        # "baz
 kubectl delete pods,services -l name=myLabel                              # name=myLabel 라벨을 가진 파드와 서비스 삭제
 kubectl delete pods,services -l name=myLabel --include-uninitialized      # 초기화되지 않은 것을 포함하여, name=myLabel 라벨을 가진 파드와 서비스 삭제
 kubectl -n my-ns delete po,svc --all                                      # 초기화되지 않은 것을 포함하여, my-ns 네임스페이스 내 모든 파드와 서비스 삭제
+# awk pattern1 또는 pattern2 와 매칭하는 모든 파드 삭제
```
Comment (seokho-son) [inline content/ko/docs/reference/kubectl/cheatsheet.md]:
```suggestion
# awk pattern1 또는 pattern2에 매칭되는 모든 파드 삭제
```

**Comment by seokho-son** [comment]
/lgtm 
@gochist Could you take a look and approve this PR?

**Comment by gochist** [comment]
@jmyung @seokho-son Thank you!
/approve

**Pull request merged**